unit-tests/algo/d2rgb/test-reproduction.cpp

```diff
@@ -149,7 +149,7 @@ int main( int argc, char * argv[] )
             {
                 // The build number is only available within Jenkins and so we have to hard-
                 // code it ><
-                std::cout << RS2_API_VERSION_STR << ".2082" << std::endl;
+                std::cout << RS2_API_VERSION_STR << ".2158" << std::endl;
```

Comment:
Can the build number be dropped ? in any event it will not reflect the actual binary version

Reply:
I update it before I release each version to validation. It's important for them.